schemas/context/deviceidentifiers.schema.json

```diff
@@ -0,0 +1,43 @@
+{
+  "meta:license": [
+    "Copyright 2017 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/xdm/context/deviceidentifiers",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "Device identifiers",
+  "type": "object",
+  "meta:extensible": true,
+  "description": "Condensed, normalized encapsulation of device identifiers.",
+  "definitions": {
+    "deviceidentifiers": {
+      "properties": {
+        "https://ns.adobe.com/experience/ecid": {
+          "title": "Adobe Experience Cloud ID",
+          "type": "string",
+          "description": "A unique identifier from Adobe Experience Cloud."
+        },
+        "https://ns.adobe.com/experience/gcmid": {
```

Comment:
This is a namespace identifier not a item name...

How about `deviceID:googleCloud`?

Reply:
the changes have been made